lib/api/WebApi.js

```diff
@@ -47,36 +47,36 @@ async function getCollisionByCollisionId(collisionId) {
   return CollisionEvent.read.validateAsync(collision);
 }
 
-async function getCollisionsByCentreline(feature, filters) {
-  const s1 = CompositeId.encode([feature]);
+async function getCollisionsByCentreline(features, filters) {
+  const s1 = CompositeId.encode(features);
   const data = { s1, ...filters };
   const options = { data };
   const collisions = await apiClient.fetch('/collisions/byCentreline', options);
   const collisionsSchema = Joi.array().items(CollisionEvent.read);
   return collisionsSchema.validateAsync(collisions);
 }
 
-async function getCollisionsByCentrelineSummary(feature, filters) {
-  const s1 = CompositeId.encode([feature]);
+async function getCollisionsByCentrelineSummary(features, filters) {
+  const s1 = CompositeId.encode(features);
   const data = { s1, ...filters };
   const options = { data };
   return apiClient.fetch('/collisions/byCentreline/summary', options);
 }
 
-async function getCollisionsByCentrelineTotal(feature) {
-  const s1 = CompositeId.encode([feature]);
+async function getCollisionsByCentrelineTotal(features) {
+  const s1 = CompositeId.encode(features);
   const data = { s1 };
   const options = { data };
   const { total } = await apiClient.fetch('/collisions/byCentreline/total', options);
   return total;
 }
 
-async function getLocationsByFeature(centrelineTypesAndIds) {
-  if (centrelineTypesAndIds.length === 0) {
+async function getLocationsByFeature(features) {
+  if (features.length === 0) {
     return new Map();
   }
-  const centrelineIds = centrelineTypesAndIds.map(({ centrelineId: id }) => id);
-  const centrelineTypes = centrelineTypesAndIds.map(({ centrelineType: type }) => type);
+  const centrelineIds = features.map(({ centrelineId: id }) => id);
```

Comment:
why don't we just change this endpoint so that it resolves a `CompositeId` to its features?